lib/matplotlib/transforms.py

```diff
@@ -110,7 +110,7 @@ class TransformNode:
     invalidated, even if 'self' is already invalid.
     """
 
-    def __init__(self, shorthand_name=None):
+    def __init__(self, shorthand_name=''):
```

Comment:
Are we sure that None will never be passed here? What about third-party libraries etc? It may require an API change note if nothing else.

Reply:
fair point.

Reply:
I agree. I will revert the change, but let me know if you want to keep it (and write a change note)

Reply:
The safe bet is to keep None. Unless a significant saving is obtained from this optimization.